formats/json/commonTest/src/kotlinx/serialization/features/PropertyExternalInitializerTest.kt

```diff
@@ -0,0 +1,49 @@
+@file:Suppress("MayBeConstant")
+
+package kotlinx.serialization.features
+
+import kotlinx.serialization.*
+import kotlinx.serialization.json.Json
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@Suppress("MemberVisibilityCanBePrivate", "unused", "ComplexRedundantLet")
+class PropertyExternalInitializerTest {
+    data class ExternalClass(
```

Comment:
I think both these classes can be in the same file, I don't see any objections

Reply:
Classes are separated by files to disable the compilation of external serializers in Legacy JS. Disable all initializer tests for the legacy JS?

Reply:
Ah, OK then, leave them as is

Reply:
Leave only one file - evaluated default value is unstable on legacy JS